client/blocks/login/style.scss

```diff
@@ -1,9 +1,13 @@
 @import "@wordpress/base-styles/breakpoints";
 @import "@wordpress/base-styles/mixins";
 @import "calypso/assets/stylesheets/p2-extends";
-
 @import "@automattic/color-studio/dist/color-variables";
 
+// Define the login container width.
+:root {
+	--login-container-width: 352px;
```

Comment:
Coming from core, I'm always careful with creating new CSS variables as they immediately become public APIs. That's less the case here of course. But I do wonder if this is the right approach. It seems like it should only be a CSS variable if it lives somewhere globally available, like inside the `base-styles` package. If it doesn't, and we want to reuse these pieces, should it instead be an SCSS variable?

Reply:
Thanks for raising the concern about global CSS variable. I have replaced it with SCSS variable in 9c170962a851b7d03cbc9a4f63ea51c061e317c6!